### ISSUE TITLE
Notification when software gets new release

### ISSUE THREAD

**Issue by cjsha** [open-ephys/bonsai-onix1-docs#141]
I don't think this could be automated because we might need to take additional actions like generating new .bonsai workflows for new operators. But it would be nice to get a notification (maybe an issue?) in the docs repo that tells us we need update the submodule pointer.

**Comment by bparks13** [comment]
There is a way, we can actually automate it from the main repo so that when a new release is tagged it generates an issue here (I think, I've never actually done that but I am pretty sure it is possible). That would be as close to automating the release as I would like to get, since I agree that there are new workflows that need to be created. 

For testing purposes, we can always have a development branch in this repo that points to the most recent commits in the main repo so that we can develop and test the workflows / extra pages before we push a new release as well.

**Comment by cjsha** [comment]
maybe we should have a branch in the bonsai-onix1 repo that gets autopopulated with the latest tags. Then the we have a check when merging to main that asks if the submodule points to the correct commit on that branch? or something like that. Idk about the exact implementation but I like the idea of having some sort of check when merging to main to make sure it's pointing to the correct commit

**Comment by ChucklesOnGitHub** [comment]
@bparks13 this is done and can be closed, right?

**Comment by bparks13** [comment]
No, this has not been accomplished yet, we still haven't decided if it's worth implementing or not, or what it would look like if we do. It's still in the Wishlist category, so we can leave it for now.
